mod/01_rating/set/type/metric_value.rb

```diff
@@ -454,7 +469,7 @@ def grade
     end
   end
 
-  view :value_link do |args|
+  view :value_link do |_args|
```

Comment:
fyi, you can also just get rid of the args here.  `view :value_link do...`.  All you need.